### DISCUSSION TITLE
Last ray of each sweep is excluded from velocity texture computation

### DISCUSSION THREAD

**Discussion by jm5791** [ARM-DOE/pyart#1465] [Q&A]
Hi all,

I am a new `pyart` user. First of all, thanks for this great tool ! 
Going through the code of the velocity texture computation util, I noticed that [the last ray of each sweep is excluded from the texture computation](https://github.com/ARM-DOE/pyart/blob/e89aca0e8cd87b567e1429421bf7d39f656b7d7f/pyart/retrieve/simple_moment_calculations.py#L291C13-L291C45). Was it done this way for some purpose I am missing or should it be fixed ?

Thanks,

**Comment by mgrover1** [comment]
@jm5791 - the last ray is kept out since we need at least 2 gates to calculate a texture, and the last value in the ray would be 0 since it is on the boundary of the dataset.
